pyscriptjs/tests/integration/support.py

```diff
@@ -243,9 +274,9 @@ def pyscript_run(self, snippet, *, extra_head="", wait_for_pyscript=True):
 # ============== Helpers and utility functions ==============
 
 
-class JsError(Exception):
+class JsErrors(Exception):
     """
-    Represent an exception which happened in JS.
+    Represent one or more exceptions which happened in JS.
```

Comment:
[typo] Represents (?)

Reply:
No, the convention in python docstrings is to prefer use the imperative than the 3rd person.
See e.g. [PEP 257])(https://peps.python.org/pep-0257/):
>The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.